# Resources

This is a repository where I collect resources that I deemed useful during my MS studies.

## Articles

https://www.nature.com/articles/nbt.2740

https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002050

http://stat.wharton.upenn.edu/~buja/sci.pdf



## Bioinformatics

[A brief history of bioinformatics](https://academic.oup.com/bib/article/20/6/1981/5066445)

[Bioinformatics on a national scale: an example from Switzerland](https://academic.oup.com/bib/article/20/2/361/3922240)

[Bioinformatics in Germany: toward a national-level infrastructure](https://academic.oup.com/bib/article/20/2/370/3738742)

[Bioinformatics in the Netherlands: the value of a nationwide community](https://academic.oup.com/bib/article/20/2/375/4159449)

[A global perspective on evolving bioinformatics and data science training needs](https://academic.oup.com/bib/article/20/2/398/4096809)

[Curated Bioinformatics Conferences](https://docs.google.com/spreadsheets/d/1_dAyUts51jgici9dy-l0IVPypUCRvmLsVXSrmVEXjoI/edit#gid=0)




## General Bioinformatics Databases

[GENCODE](https://www.gencodegenes.org/#): identify and classify all gene features in the human and mouse genomes 

[NONCODE](http://www.noncode.org/): knowledge database dedicated to non-coding RNAs


## lncRNA Tools

[ExInator](https://github.com/alanzos/ExInAtor): designed to detect cancer driver long non-coding RNA genes


[lncATLAS](https://lncatlas.crg.eu/): Find the subcellular localization of your lncRNA of interest. Contains annotated lncRNA data from GENCODE.



## File Extensions

### .maf Mutation Annotation Format

The Mutation Annotation Format (MAF) file is a tab-delimited text file that contains information about the mutations detected in the sequencing project. It lists one mutation per row, and the columns (named in the header row) report several pieces of information for each mutation.


## Platforms

[GenePattern](https://www.genepattern.org/#): GenePattern provides access to a broad array of computational methods used to analyze genomic data in a graphical user interface. 

[GenePattern Notebooks](https://notebook.genepattern.org/): Jupyter Notebooks repo containing many pipelines.

# Cancer Research

## Cancer Mutation Databases

[COSMIC Cancer Database](https://cancer.sanger.ac.uk/cosmic/download)

[TCGA Cancer Database](https://portal.gdc.cancer.gov/)


## Pipelines for identification of protein coding driver genes

[MutSigCV](https://www.genepattern.org/modules/docs/MutSigCV): Identifies genes that are significantly mutated in cancer genomes

[OncoDriveFML](https://bbglab.irbbarcelona.org/oncodrivefml/home): a method designed to analyze the pattern of somatic mutations across tumors in both coding and non-coding genomic regions to identify signals of positive selection, and therefore, their involvement in tumorigenesis.

[OncoDriveCLUSTL](https://bbglab.irbbarcelona.org/oncodriveclustl/home): nucleotide sequence-based clustering algorithm to detect cancer drivers in genomic regions.


# Useful Stuff


[Create a timeline from a google spreadsheet](https://timeline.knightlab.com/#make)

[Free & Open Source Website Builder](https://wowchemy.com/)

[Academic Phrase Bank](https://www.phrasebank.manchester.ac.uk/)

[Ten ways to find Open Access articles](https://alelazic.blogspot.com/2021/11/ten-ways-to-find-open-access-articles.html)
[]()
[]()
